exp/api/v1alpha3/groupversion_info.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
```

Comment:
We should probably remove the v1alpha3 folder altogether?

Reply:
I guess that makes sense. I was thinking it might be easier to copy the existing CRDs over to the new API group in v1alpha3 and then convert them to v1alpha4 than having to do the API group rename _and_ conversion manually as a user, WDYT?

Reply:
Got it, I am not 100% sure what would be the best way, I am ok keeping them if we all prefer